packages/appstore/src/components/account-transfer-modal/account-transfer-modal.tsx

```diff
@@ -24,13 +25,18 @@ const AccountTransferModal = observer(({ is_modal_open, toggleModal }: TAccountT
     const history = useHistory();
 
     React.useEffect(() => {
+        if (is_modal_open) setActiveTab('account_transfer');
+
         return () => {
-            setShouldSwitchAccount(false);
-            setSelectedAccount({});
-            closeModal();
+            if (is_modal_open) {
```

Comment:
here we do the cleanup when the user is closing the modal
!!!
the condition is right, there should be `is_modal_open`, not `!is_modal_open`, because  cleanup is firing in the same lifecycle in which `is_modal_open` is still `true`.